release.sh

```diff
@@ -23,9 +24,9 @@ if [ ! -z $ms_version_suffix ]; then
 fi
 
 echo "#"
-echo "# make sure:"
+echo "# Make sure:"
 echo "# - you are on branch-$ms_version_major-$ms_version_minor"
-echo "# - you have edited HISTORY.TXT with changes related to this release"
+echo "# - you have edited MIGRATION_GUIDE.txt with changes related to this release"
```

Comment:
this line should definitely mention HISTORY.TXT   

if you want, maybe you can change it to "...you have edited HISTORY.TXT and possibly MIGRATION_GUIDE.txt with changes related to this release..."

Reply:
oh! I was referring to the script in /mapserver    sorry!  please ignore my comment :)